lib/faraday/adapter/net_http_persistent.rb

```diff
@@ -184,6 +184,7 @@ def request_with_wrapped_block(http, env)
         ssl_version: :version,
         min_version: :min_version,
         max_version: :max_version
+        verify_hostname: :verify_hostname
```

Comment:
Faraday [defaults this option to true](https://github.com/lostisland/faraday/blob/3170e7df6fe40b49a2f406c0aad503a6d9e2aa12/docs/customization/ssl-options.md), so we don't need to hardcode any default in the adapter.